1.0/docs/devguide/registering-elements.md

```diff
@@ -228,53 +228,84 @@ local DOM when the element is constructed.
     }
 
 **Note:** This example uses [Automatic node finding](local-dom.html#node-finding) to
-access a local DOM element. 
+access a local DOM element.
 {: .alert .alert-info }
 
 Within a given tree, `ready` is generally called in _document order_, but you should not
-rely on the ordering of initialization callbacks between sibling elements, or between 
+rely on the ordering of initialization callbacks between sibling elements, or between
 a host element and its light DOM children.
 
+### Initialization order and timing {#initialization-order}
 
-### Initialization order {#initialization-order}
+The element's basic initialization order for a given element is:
 
-The element's basic initialization order is:
+- `created` callback   local DOM initialized (This means that **local DOM**
+- children are created, their property values are set as specified in the
+- template, and `ready()` has been called on them) `ready` callback
+- [`factoryImpl` callback](#custom-constructor) `attached` callback
 
-- `created` callback  
-- local DOM initialized 
-- `ready` callback
-- [`factoryImpl` callback](#custom-constructor)
-- `attached` callback
+Note that while the life cycle callbacks listed above will be called in the
+described order for any given element, the  **initialization timing between
+elements may vary** depending on whether or not the browser includes native
+support for web components.
 
-Note that the **initialization order may vary** depending on whether or not the
-browser includes native support for web components. In particular, there are no
-guarantees with regard to initialization timing between **sibling elements** or
-between **parents and light DOM children**. You should not rely on observed
-timing to be identical across browsers, except as noted below.
+#### Initialization timing for light DOM children
 
-For a given element:
+As far as initialization timing of light DOM children, there are no guarantees
+at all; it can generally be expected them to be ready in document order, and
```

Comment:
"it can generally be expected them to be ready in document order, and that they will be _usually_ initialized after their parents. " => in general elements are initialized in document order, so children are _usually_ initialized after their parents.